cmd/telegraf/agent.conf

```
@@ -53,13 +53,13 @@
   ## Log only error level messages.
   # quiet = false
 
-  ## Log target controls the destination for logs and can be one of "file",
-  ## "stderr" or, on Windows, "eventlog".  When set to "file", the output file
-  ## is determined by the "logfile" setting.
+  ## Log target controls the destination for logs and can be one of "text",
+  ## "structured" or, on Windows, "eventlog". The output file (if any) is
+  ## determined by the "logfile" setting.
   # logtarget = "file"
```

Needs to be updated to reflect the new default?
```suggestion
  # logtarget = "text"
```